web/src/reducers/budget.test.js

```diff
@@ -287,7 +287,7 @@ describe('budget reducer', () => {
               },
               subtotal: {
                 state: { dollars: 3060, percent: 1 },
-                contractors: { dollars: 1800, percent: 1.0000000000000002 },
+                contractors: { dollars: 1800, percent: 1 },
```

Comment:
I was hoping to do this refactor without changing the test file at all, but I feel okay changing this wild floating point values.

Reply:
yesssssss